src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {

```

Comment:
The class name 'Drawable' sounds like a verb. According to best practices, interfaces should not be named as verbs. Consider renaming it to something like 'DrawableFigure' to make it clear that it's an interface representing a drawable figure.